app/src/main/java/com/example/ncov19traking/ui/countries/CountriesViewModel.kt

```diff
@@ -3,14 +3,17 @@ package com.example.ncov19traking.ui.countries
 import android.app.Application
 import androidx.lifecycle.AndroidViewModel
 import androidx.lifecycle.liveData
+import com.example.ncov19traking.data.NCoVDataBase
 import com.example.ncov19traking.data.NCoVRepository
 import kotlinx.coroutines.Dispatchers
 
-class CountriesViewModel(application: Application) : AndroidViewModel(application) {
+class CountriesViewModel(context: Application) : AndroidViewModel(context) {
```

Comment:
Do you need to receive the `context` as an `Application`, you could also change it to be `Context`

Reply:
As far as I know its mandatory, because AndroidViewModel its a class of Android Jetpack which need and only an Application by constructor